src/specialized/mod.rs

```diff
@@ -6,7 +6,7 @@ cfg_if! {
     ))] {
         mod pclmulqdq;
         pub use self::pclmulqdq::State;
-    } else if #[cfg(all(feature = "nightly", target_arch = "aarch64"))] {
+    } else if #[cfg(target_arch = "aarch64")] {
```

Comment:
it's still guarded by `crc` feature flag, so I'd keep it.

Reply:
It's best not to check for the `crc` target feature here. [The specialized `State` for AArch64 already ensures that these instructions are supported](https://github.com/srijs/rust-crc32fast/blob/479ecdf0174dd3a0f7d48b2f66a386d8d2369963/src/specialized/aarch64.rs#L9-L29), either at runtime or through target features, depending on whether the `std` feature is enabled. If support is lacking in either case, a [fallback to the baseline implementation is done](https://github.com/srijs/rust-crc32fast/blob/479ecdf0174dd3a0f7d48b2f66a386d8d2369963/src/lib.rs#L104-L107). So, adding a target feature check here would render runtime detection useless for targeting AArch64 CPUs with mixed support for these instructions.